src/directives/ctr-input.ts

```diff
@@ -157,6 +158,19 @@ export class CtrInput {
         }
     }
 
+    @HostListener("click", ["$event"])
+    public onClick(event: any) {
+        this.el.nativeElement.select();
```

Comment:
I got it your change and I agree. I already approved your PR.

What do we need to have my changes merged here?